src/Modules/MotdModule/MotdPermissions.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.ComponentModel;
+using EvoSC.Common.Permissions.Attributes;
+using EvoSC.Common.Util.EnumIdentifier;
+
+namespace EvoSC.Modules.Official.MotdModule;
+
+[PermissionGroup]
+[Identifier(Name = "MotdPermissions")]
```

Comment:
This is actually redundant because it will just take the name of the enum automatically.